README.md

```
@@ -1,4 +1,4 @@
-![Tests Passing](https://github.com/NicolaWealth/rate_limit/actions/workflows/auto_test_main_badge.yml/badge.svg)
+![Tests Passing](https://github.com/NicolaWealth/rate_limit/actions/workflows/test.yml/badge.svg)
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)